RubberduckTests/QuickFixes/UseSetKeywordForObjectAssignmentQuickFixTests.cs

```diff
@@ -63,5 +110,19 @@ protected override IQuickFix QuickFix(RubberduckParserState state)
         {
             return new UseSetKeywordForObjectAssignmentQuickFix();
         }
+
+        protected override IVBE TestVbe(string code, out IVBComponent component)
```

Comment:
What this method does should be accomplishable through `MockVbe.BuildFromSingleStandardModule`. Do we need to merge these?